src/hardware/input/mouse_interfaces.cpp

```diff
@@ -389,9 +389,9 @@ uint16_t MouseInterface::GetRate() const
 	return rate_hz;
 }
 
-void MouseInterface::NotifyInterfaceRate(const uint16_t new_rate_hz)
+void MouseInterface::NotifyInterfaceRate(const uint16_t in_rate_hz)
 {
-	interface_rate_hz = new_rate_hz;
+	interface_rate_hz = in_rate_hz;
```

Comment:
Maybe just `rate_hz`? "in" can be be interface... or "input" perhaps?

Reply:
Brought back the `new_` prefix.